Services/Certificate/classes/Helper/ilCertificateExerciseMembersHelper.php

```diff
@@ -6,12 +6,7 @@
  */
 class ilCertificateExerciseMembersHelper
 {
-    /**
-     * @param int $objectId
-     * @param int $userId
-     * @return mixed
-     */
-    public function lookUpStatus(int $objectId, int $userId)
+    public function lookUpStatus(int $objectId, int $userId) : ?string
```

Comment:
@mBeym Can you please check the `Consumers` of this call? Maybe we have to handle the possible `null` value.

Reply:
value returned by lookUpStatus is already checked in the consumers.

Reply:
Hmmm, that's not true IMO:

See: \ilExercisePlaceholderValues::getPlaceholderValues If `$status` is `NULL`, this will cause an error or lead to unexpected behaviour because of `$this->language->txt('exc_' . $status)`.

Reply:
Should be fixed now.

ilExercisePlaceholderValues::99